api-tests/src/steps/cri-steps.ts

```diff
@@ -84,3 +84,29 @@ When(
     );
   },
 );
+
+When(
+  /^I submit '([\w-]+)' details to the (async )?CRI stub with '([\w-]+)' CI to mitigate$/,
```

Comment:
I might rephrase this slightly, as it reads a bit strangely, perhaps:
`I submit 'some' details to the CRI stub that mitigate the 'ALTERNATE_DOC' CI`

You could consider combining this with the existing step as well, but it might be easier to follow if it's kept separate!

Reply:
Agreed! I tried combining them but the regex seemed messy with extra logic to handle the nested regex groups so I thought it might be better to keep this cimit step separate from regular "submit details" steps

Reply:
Oh I didn't know non-capturing groups were a thing (still a regex newbie :')) which makes it much simpler but it might still be clearer to separate this as a separate cimit-specific step?

Reply:
I'm not sure it's ever possible to become a regex expert 😅